README.md

```diff
@@ -2,7 +2,7 @@
 
 Welcome to the Fabulously Optimized wiki!
 
-**[📜 List of included mods](https://github.com/Fabulously-Optimized/fabulously-optimized/blob/main/INCLUDED-MODS.md)** | **[❗Chat Reporting FAQ](en-us/chat-reporting-faq.md)**
+**[📜 List of included mods](https://github.com/Fabulously-Optimized/fabulously-optimized/blob/main/INCLUDED-MODS.md)** | **[❗Chat Reporting FAQ](en-us/chat-reporting-faq.md)** | **[❓FAQ](en-us/faq.md)**
```

Comment:
put it as the first item of "More info about" instead
name should be "🍦Vanilla launcher FAQ"